Sources/PointFree/Blog/BlogPost.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import Prelude
+
+public struct BlogPost {
+  public typealias Id = Tagged<BlogPost, Int>
+
+  public private(set) var blurb: String
+  public private(set) var contentBlocks: [Episode.TranscriptBlock]
```

Comment:
reusing transcript blocks for blog content blocks even tho timestamps don't make sense. let's just always use `nil` for now. we can refactor at a later time.